feature-chat/src/main/res/layout/fragment_channel_list.xml

```diff
@@ -29,7 +29,7 @@
         android:layout_width="match_parent"
         android:layout_height="match_parent">
 
-        <io.getstream.chat.android.ui.channel.list.header.ChannelListHeaderView
+        <io.getstream.chat.android.ui.feature.channels.header.ChannelListHeaderView
```

Comment:
Some Views have been moved to other package names to keep the parity with the `compose` artifact